docs/deploy-development-rp.md

```diff
@@ -4,6 +4,9 @@
 
 1. Your development environment is prepared according to the steps outlined in [Prepare Your Dev Environment](./prepare-your-dev-environment.md)
 
+## There is a script to execute the ARO-RP locally
```

Comment:
Title is awkward in this context. Maybe we can do something like:
```suggestion
## Automatically run local RP
```
I also think that this section would be better between the `Run the RP and create a cluster` and `Make Admin-Action API call(s) to a running local-rp` sections, for reference by those who have gone through this process before. To someone who is new to this process IMO it is better to go through the steps manually to know how the process works.

Reply:
Done. Tks!
BTW, "IMO it is better to go through the steps manually to know how the process works." I  100% agree on that, that's why that is mentioned both in the .md file and in the description of the shell script (top of the file).
😃